packages/orchestration/test/examples/sendAnywhere.test.ts

```diff
@@ -0,0 +1,153 @@
+import { commonSetup } from '../supports.js';
+
+const dirname = path.dirname(new URL(import.meta.url).pathname);
+
+const contractName = 'sendAnywhere';
+const contractFile = `${dirname}/../../src/examples/${contractName}.contract.js`;
+type StartFn =
+  typeof import('../../src/examples/sendAnywhere.contract.js').start;
+
+const chainInfoDefaults = {
+  connections: {},
+  allowedMessages: [],
+  allowedQueries: [],
+  ibcHooksEnabled: false,
+  icaEnabled: false,
+  icqEnabled: false,
+  pfmEnabled: false,
+};
+
+const connectionDefaults = {
+  versions: [
+    {
+      identifier: '1',
+      features: ['ORDER_ORDERED', 'ORDER_UNORDERED'],
+    },
+  ],
+  delay_period: 0n,
+};
+
+const txChannelDefaults = {
+  counterPartyPortId: 'transfer',
+  version: 'ics20-1',
+  portId: 'transfer',
+  ordering: 1, // ORDER_UNORDERED
+  state: 3, // STATE_OPEN
+};
+
+test('send using arbitrary chain info', async t => {
+  t.log('bootstrap, orchestration core-eval');
+  const {
+    bootstrap,
+    commonPrivateArgs,
+    brands: { ist },
+    utils: { inspectLocalBridge },
+  } = await commonSetup(t);
+
+  const { rootZone: zone, board, marshaller } = bootstrap;
+  const timerBrand = await E(bootstrap.timer).getTimerBrand();
+
+  const { zoe, bundleAndInstall } = await setUpZoeForTest();
+
+  t.log('contract coreEval', contractName);
+
+  const installation: Installation<StartFn> =
+    await bundleAndInstall(contractFile);
+
+  const { instance, creatorFacet } = await E(zoe).startInstance(
+    installation,
+    { Stable: ist.issuer },
+    {},
+    {
+      timerBrand,
+      board,
+      marshaller,
+      ...commonPrivateArgs,
+      storageNode: await E(bootstrap.storage.rootNode).makeChildNode(
+        contractName,
+      ),
+    },
+  );
+
+  t.log('admin adds hot new chain to contract');
+  const hotChainInfo = harden({
+    chainId: 'hot-new-chain-0',
+    stakingTokens: [{ denom: 'uhot' }],
+    ...chainInfoDefaults,
+  }) as CosmosChainInfo;
+  const agoricToHotConnection = {
+    ...connectionDefaults,
+    id: 'connection-1',
+    client_id: '07-tendermint-1',
+    state: 3, // STATE_OPEN
+    counterparty: {
+      client_id: '07-tendermint-2109',
+      connection_id: 'connection-1649',
+      prefix: {
+        key_prefix: 'aWJj',
+      },
+    },
+    transferChannel: {
+      counterPartyChannelId: 'channel-1',
+      channelId: 'channel-0',
+      ...txChannelDefaults,
+    },
+  } as IBCConnectionInfo;
+  t.log('add chain using creatorFacet', hotChainInfo.chainId);
+  const chainKey = await E(creatorFacet).addChain(
+    hotChainInfo,
+    agoricToHotConnection,
+  );
+
+  t.log('client uses contract to send to hot new chain');
+  const publicFacet = await E(zoe).getPublicFacet(instance);
+  const inv = E(publicFacet).makeSendInvitation();
+
+  const amt = await E(zoe).getInvitationDetails(inv);
+  t.is(amt.description, 'send');
+
+  const anAmt = ist.make(20n);
+  const Send = ist.mint.mintPayment(anAmt);
```

Comment:
use `pourPayment` to get balance tracking (vbank not aware of minting from thin air )

this or soon we should take `mint` out of the withAmountUtils that comes out of setup